activejob/lib/active_job/queue_adapters/sidekiq_adapter.rb

```diff
@@ -18,21 +18,17 @@ module QueueAdapters
     #   Rails.application.config.active_job.queue_adapter = :sidekiq
     class SidekiqAdapter
       def enqueue(job) # :nodoc:
-        # Sidekiq::Client does not support symbols as keys
```

Comment:
Since `set` only works after sidekiq 4.1 it is better to add a `gem "sidekiq", ">= 4.1.0"` in the top of the file.

Reply:
4.1 was released in Jan 2016. Do you think anyone is using an ancient Sidekiq version with edge Rails?

Reply:
Not really, but we never know 😂. I'm ok with this patch though. 2016 is indeed too ancient.

Reply:
To me that's an argument for adding the `gem`: we're okay with adopting it as a minimum version, but we should communicate that fact just in case someone is [perhaps accidentally] doing something weird, rather than leave them to fail on an opaque NoMethodError.

Reply:
Yeah. I'm already adding the gem call. We do for all the database adapters, we should do here as well.